package/src/contexts/messageInputContext/MessageInputContext.tsx

```diff
@@ -1000,34 +990,63 @@ export const MessageInputProvider = <
     type?: string;
     uri?: string;
   }) => {
-    const id = generateRandomId();
-    const mimeType = lookup(file.name);
-    const newFile = {
+    const id: string = generateRandomId();
+    const mimeType: string | boolean = lookup(file.name);
+
+    const isBlockedFile: boolean | undefined = blockedFileExtensionTypes?.some(
+      (fileExtensionType: string) => file.name?.includes(fileExtensionType),
+    );
+    const isBlockedMime: boolean | undefined = blockedFileMimeTypes?.some((mimeType: string) =>
```

Comment:
I'd keep it as `isBlockedMimeType` since `MIME` by itself really isn't a thing/isn't commonly used. We also use `mimeType` elsewhere here

Reply:
✅

Reply:
I think I'll rename these variables to `isBlockedFileMimeType` and `isBlockedFileExtension`. Giving them their names that describe where they are being used and if they are blocked file extensions or  blocked file mime types.

Reply:
✅